pkg/clients/clients_test.go

```
@@ -28,15 +28,19 @@ var _ = Describe("Client", func() {
 	When("A clientset is requested with no kubeconfig", func() {
 		It("should panic", func() {
```

It should no longer panic, it should return an error.